docs/3.x/routing.md

```diff
@@ -148,3 +148,22 @@ If you access `http://my-project.test/blog/archive/2018`, your `blog/_archive.tw
 ```twig
 <h1>Blog Entries from {{ year }}</h1>
 ```
+
+### Accessing Named Parameters in your Controllers
+
+To access named parameters in your contollers, you will need to add the parameter(s) to your controller's action definition.
+
+For example, with this URL rule:
+
+```php
+'blog/archive/<year:\d{4}>' => 'controller/action/foo',
+```
+
+You controller method would be:
```

Comment:
Should be “_Your_ controller method” but otherwise good to go!

Reply:
I'm just going to go hide now.

Reply:
Oh I didn’t mean to be rude—I’ll take care of it. Again, thank you!